data/json/mapgen/cosmics_additions/GardenBotanical.json

```diff
@@ -196,7 +197,8 @@
         { "signage": "CATTAIL", "x": 4, "y": 14 },
         { "signage": "DATURA", "x": 4, "y": 19 },
         { "signage": "FUNGUS", "x": 9, "y": 14 },
-        { "signage": "CACTUS", "x": 9, "y": 19 }
+        { "signage": "CACTUS", "x": 9, "y": 19 },
+        { "signage": "KENTUCKY COFFEE", "x": 15, "y": 3 }
```

Comment:
I'd suggest making the sign just read "COFFEE" as to fit with the rest of the signs being all one word (APPLE, PINE, etc.).

Reply:
Ok, fair enough. I didn't know it was difference and had to look it up to find out its a completely different species.